### DISCUSSION TITLE
Android: Can I prevent the GPS connection from being interrupted when I put the cell phone in my pocket?

### DISCUSSION THREAD

**Discussion by PetersOtto** [organicmaps/organicmaps#8070] [Q&A] [answered]
Hello!

I use Organic Map for hiking. I like the program very much! 
I insert the tracks as `kml ` and walk the displayed route. I look at my cell phone and note the next few meters (or more). Then I switch off the display and put it in my pocket.

The next time I look at my cell phone, it searches for the GPS connection again. That always takes a while.

Can I prevent the GPS connection from being interrupted when I put the cell phone in my pocket?

- Android 12 /e/ murena
- Orcanic Map: 2024.03.31-8-FDroid

**Comment by hb0nd** [comment, marked as answer]
As a temporary workaround, you can use this: https://f-droid.org/packages/com.github.mrrar.gps_locker/

**Reply by PetersOtto** [reply]
Oh, cool. Thanks for the quick reply!

**Reply by hb0nd** [reply]
no worries :)

the full fix is: https://github.com/organicmaps/organicmaps/issues/1360, but that's probably a way off
